{{cookiecutter.project_name}}/next_steps.md

```diff
@@ -0,0 +1,36 @@
+# Next steps
+
+## Put the generated files under version control
+
+Once your Python package is created, put it under [version
+control](https://guide.esciencecenter.nl/#/best_practices/version_control) using
+[git](http://git-scm.com/) and [GitHub](https://github.com/).
+
+```shell
+cd {{ cookiecutter.project_name }}
+git init
+git add --all
+git commit -m "first commit"
+git branch -M main
+git remote add origin {{ cookiecutter.repository }}
```

Comment:
You will get a https remote instead of a ssh. Which I dont like, I used the instructions on GitHub post repo create page instead.

However [GH recommends HTTPS](https://docs.github.com/en/github/getting-started-with-github/set-up-git) so no problem.

Reply:
ok